globus-requirements.txt

```diff
@@ -0,0 +1,6 @@
+# extra requirements for globus
+-r ./requirements.txt
+requests>=2.0.0,<3.0.0
+six==1.10.0
+python-jose
+pycrypto
```

Comment:
OK, then I think this requirements.txt can be updated to have just those:

```
-r ./requirements.txt
python-jose
globus_sdk
```

Any other dependencies globus_sdk may have should be specified by that package.